crates/test-utils/src/messages.rs

```diff
@@ -326,6 +329,7 @@ pub fn make_publish_basics_transaction(gas_object: ObjectRef) -> VerifiedTransac
         sender,
         gas_object,
         all_module_bytes,
+        vec![],
```

Comment:
Note, this depends on `Sui` and `MoveStdlib`.

Reply:
(Turns out this function is completely unused, as are a couple around it, so I'll just get rid of them)